src/main/java/org/ow2/proactive/catalog/service/WorkflowXmlManipulator.java

```diff
@@ -111,6 +133,10 @@ private Element replaceJobName(Element element, String jobName) {
         return replaceAttributeValue(element, "name", jobName);
     }
 
+    private Element replaceOrAddProjectName(Element rootElement, String projectName) {
```

Comment:
Check that projectName is an empty space :)

Reply:
projectName can be empty string